### PR TITLE
INDY-902: Indy Node tests integration with Indy SDK

### PR DESCRIPTION
**Pull request by ArtObr**

### CODE REVIEW

**Comment by dsurnin** [comment]
test this please

File: indy_node/test/attrib_txn/test_nym_attrib.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+import base58
+import libnacl.public
+import pytest
+
+from plenum.common.constants import ENC, TXN_ID, TARGET_NYM, \
+    TXN_TYPE, NONCE, STEWARD_STRING
+from plenum.common.exceptions import RequestRejectedException
+from plenum.common.signer_did import DidSigner
+from plenum.common.util import adict, randomString
+from indy_client.client.wallet.attribute import Attribute, LedgerStore
```
Comment (ashcherbakov) [inline indy_node/test/attrib_txn/test_nym_attrib.py]: No code from indy-client can be used anymore

Comment (dsurnin) [reply indy_node/test/attrib_txn/test_nym_attrib.py]: this is intermediate PR
the other code will be deleted with next PRs

Comment (ashcherbakov) [reply indy_node/test/attrib_txn/test_nym_attrib.py]: ok

**Pull request merged**